src/Splitter.story.lua

```diff
@@ -0,0 +1,93 @@
+local Packages = script.Parent.Parent
+local Roact = require(Packages.Roact)
+
+local Constants = require(script.Parent.Constants)
+
+local Splitter = require(script.Parent.Splitter)
+local PluginProvider = require(script.Parent.PluginProvider)
+
+local Label = require(script.Parent.Label)
+local ScrollFrame = require(script.Parent.ScrollFrame)
+local Dropdown = require(script.Parent.Dropdown)
+
+local Wrapper = Roact.Component:extend("Wrapper")
+
+function Wrapper:init()
+	self:setState({
+		Alpha0 = 0.5,
+		Alpha1 = 0.5,
+		Alpha2 = 0.5,
+	})
+end
+
+function Wrapper:render()
+	local scrollContents = {}
+	for i = 1, 20 do
+		scrollContents[i] = Roact.createElement(Label, {
+			Size = UDim2.new(1, 0, 0, 20),
+			Text = "Item " .. i,
+			LayoutOrder = i,
+		})
+	end
+
+	return Roact.createElement(Splitter, {
+		Alpha = self.state.Alpha0,
+		OnAlphaChanged = function(newAlpha)
+			self:setState({ Alpha0 = newAlpha })
+		end,
+		Orientation = Constants.SplitterOrientation.Vertical,
+	}, {
+		[1] = Roact.createElement(ScrollFrame, {
```

Comment:
Question, why the explicit indexes here? They can be omitted and still work fine.

Reply:
I think it illustrates how Splitter remaps children more clearly than without the explicit indices. This should really be covered by good docs instead.

Reply:
Good documentation would be better. It's also possible to give them a string name like `First` and `Second`, too.